med-web-app-front/src/services/pipelinejob.service.js

```diff
@@ -12,6 +12,10 @@ class PipelineJobService {
     sendRequestForPipelineJob(username, pipelineId, fileId) {
         return axios.post(API_URL + username, {pipelineId, fileId},{ headers: authHeader() });
     }
+
+    deletePipelineJob(username, pipelineJobId, fileId) {
+        return axios.delete(API_URL + username + "/" + pipelineJobId + "/" + fileId,{ headers: authHeader() });

```

Comment:
Codacy has a fix for the issue: [Replace `········return·axios.delete(API_URL·+·username·+·"/"·+·pipelineJobId·+·"/"·+·fileId,{·headers:·authHeader()·});␍` with `····return·axios.delete(⏎······API_URL·+·username·+·"/"·+·pipelineJobId·+·"/"·+·fileId,⏎······{·headers:·authHeader()·}⏎····);`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534713)
```suggestion
    return axios.delete(
```